client/my-sites/plans/plan-overview/plan-status/index.jsx

```diff
@@ -41,7 +65,9 @@ const PlanStatus = React.createClass( {
 		return (
 			<div className="plan-status">
 				<CompactCard className="plan-status__info">
-					<div className={ iconClasses } />
+					<div className={ iconClasses }>
+						{ isInGracePeriod( plan ) && <Gridicon icon="notice" /> }
```

Comment:
Good idea. The data we need here to determine if a site is in the grace period isn't returned by the `/me/sites` endpoint, so this will take some thought. I opened an issue for it here: #1848